library/src/main/java/com/getstream/sdk/chat/enums/MessageInputType.java

```diff
@@ -1,5 +1,16 @@
 package com.getstream.sdk.chat.enums;
 
 public enum  MessageInputType {
-    EDIT_MESSAGE, ADD_FILE, UPLOAD_MEDIA, UPLOAD_CAMERA, UPLOAD_FILE, COMMAND, MENTION
+    EDIT_MESSAGE("Edit a message"),
```

Comment:
I don't think this is an improvement to previous code; if anything it is worse since it makes it harder to customize the title

Reply:
For customizing it, we won't need MessageInputType enum as well, I think.